triggers/runner/src/main/scala/com/digitalasset/daml/lf/engine/trigger/Runner.scala

```diff
@@ -577,7 +593,30 @@ object Runner extends StrictLogging {
     def unapply(v: SPAP): Some[SPAP] = Some(v)
   }
 
-  private def alterF[K, V, F[_]: Functor](m: Map[K, V], k: K)(
+  /** Like `CommandRetryFlow` but with no notion of ledger time, and with
```

Comment:
Looking at `CommandRetryFlow` it is structured pretty differently and pushes retries back through the flow. Is there a reason why the two are so different? What is stopping us from pushing retries back through the stream?

Reply:
See commit message 7d969a67f42d0dfc42749496c7d1040d4cca56ad for an explanation.